### DISCUSSION TITLE
What is the smallest monitor resolution that GO supports?

### DISCUSSION THREAD

**Discussion by willeke1234** [GrandOrgue/grandorgue#2008] [Developers]
I'm playing around with the MIDI Objects dialog to see what wxWidgets can do. One thing leads to another and the dialog is growing. It looks ok on my 24" monitor, but of course it should work on smaller monitors too.

What is the smallest monitor resolution that GO supports? Or what is the smallest size a dialog should support? Are dialogs largest on Linux?

**Comment by larspalo** [comment]
> What is the smallest monitor resolution that GO supports?

Supports? Rather ask what is the smallest conveniently usable resolution possible. I've recently used GO on a very old netbook Atom where max resolution is 1024x600. Usable? Sure, but not very nice or convenient to use! One needs to drag some dialogs around to see/access the lower content of some of the dialogs as not all of them have scrollbars to access OK buttons and such... But I do think that not too much more is absolutely necessary. If I remember correctly a 1366x768 monitor was more or less without issues last I tried it. Most monitors nowadays are capable of much higher resolution though, I just looked at a fairly large company/supplier for consumers here in Sweden and nothing lower than 1920x1080 is actually listed anymore on any monitor sold...

**Reply by hnb2907** [reply]
Hi,

I've got a pair of 10" touchscreens 1280x800.  For my application, they physically sit well on the console, either side of the keyboards, and are ergonomically good.

All of the GO dialogs seem to be ok at this resolution, but when you're are accustomed to using larger monitors which higher resolution, they do feel cramped.  The majority of the time, the dialog windows are not being used.

I would suggest that you mimic the size of your window containing the stops on another monitor, especially if you're planning to use small touchscreens (and you haven't bought them yet),   Get the window to be the equivalent size on another monitor, and make sure that the stops look a reasonable size and spacing, that you could sensibly operate them with your finger.

Here's an example of my 2x stop windows, 1 on each 10" monitor.  The stops are just big enough to press on the touchscreen.  Any smaller, and they'd be really annoying to "touch" accurately while playing.  The small round buttons are tricky to use, but are mapped to physical buttons on the console, so are never used on the touchscreens.  :)

![Screenshot at 2024-09-14 22-24-24](https://github.com/user-attachments/assets/fc357110-1238-4c96-94ca-ba6efd006384)
![Screenshot at 2024-09-14 22-22-49](https://github.com/user-attachments/assets/03afbdc8-1020-419f-855e-d49dd59eb2f8)


Cheers,
Chris.

**Reply by hnb2907** [reply]
By the way, this is on Ubuntu Studio with Gnome.

**Reply by oleg68** [reply]
@willeke1234 what problems do you have with small monitors? Even a dialog window does not fit the screen, the scrollbars must appear.

Possible, they do not appear in MacOs, do they?

**Reply by willeke1234** [reply]
@oleg68 The scrollbars do appear, but double scrollbars don't work well, see #1972. It can be inconvenient if part of the GUI is invisible. My goal is to make GO easier to use.
